### DISCUSSION TITLE
Download a complete album at once

### DISCUSSION THREAD

**Discussion by awesomeinsight** [photoview/photoview#605] [Q&A and Troubleshooting]
Hi @ all!

First of all - thanks for the great work!
I have one question: Is it possible to download a complete album at once.
I only see the download option on single images but not on album level.
Is it configurable or even possible?

Thanks in advance!

**Comment by viktorstrate** [comment]
Hi @awesomeinsight that is possible, by going to the sidebar for a particular album, you do so by clicking on the gear icon next to the album name. From there you should see a downloads section.

This feature was added in [v2.3.7](https://github.com/photoview/photoview/releases/tag/v2.3.7)

**Reply by awesomeinsight** [reply]
Hi @viktorstrate! 
Thanks for your quick reply.
Works like a charm!

**Comment by GyroGearl00se** [comment]
Any chance download all button can be also added to a album via shared link ?
